traffic_ops/traffic_ops_golang/crconfig/crconfig.go

```diff
@@ -30,7 +30,7 @@ import (
 )
 
 // Make creates and returns the CRConfig from the database.
-func Make(tx *sql.Tx, cdn, user, toHost, reqPath, toVersion string, useClientReqHost bool, emulateOldPath bool) (*tc.CRConfig, error) {
+func Make(tx *sql.Tx, cdn, user, toHost, toVersion string, useClientReqHost bool, emulateOldPath bool) (*tc.CRConfig, error) {
```

Comment:
Either way we'd need `CRConfig.Stats.TMPath`; `CRConfigEmulateOldPath` only determined whether or not to insert a legacy path into that field. It would still be populated otherwise, just with a different path.

I believe the "major version" that's talking about is the major ATC version, since historically snapshots have been un-versioned.

Reply:
Oh yeah, it can be separate from this PR. I just figured if we're gutting a field and it's dependent upon another deprecated field that _can_ be removed then we can just keep cleaning. Anyways, another GH issue is just fine for me here.